src/lib/updateGrantsGrantees.js

```diff
@@ -96,27 +98,45 @@ export async function processFiles() {
  *
  * Note - file download needs to happen in deployed environments
  */
-export default async function updateGrantsGrantees() {
-  try {
-    if (process.env.NODE_ENV === 'production') {
-      const response = await axios(process.env.HSES_DATA_FILE_URL, {
-        method: 'get',
-        url: process.env.HSES_DATA_FILE_URL,
-        responseType: 'stream',
-        auth: {
-          username: process.env.HSES_DATA_USERNAME,
-          password: process.env.HSES_DATA_PASSWORD,
-        },
+export default async function updateGrantsGrantees(_processFiles = processFiles) {
```

Comment:
`_processFiles = processFiles` using dependency injection to more easily pass a mock function in from for just testing this function. The other option was to break processFiles out into its own file to more easily mock it with jest.